include/oneapi/dpl/async

```diff
@@ -0,0 +1,45 @@
+// -*- C++ -*-
+//===-- async -------------------------------------------------------------===//
+//
+// Copyright (C) 2020 Intel Corporation
```

Comment:
According to the changes in #94 year is not needed anymore in the copyright. The PR also contains the script that can be applied to all files

Reply:
done